docker/README.md

```diff
@@ -64,21 +64,21 @@ bitcoin into.
 $ export NETWORK="simnet" 
 
 # Run the "Alice" container and log into it:
-$ docker-compose run -d --name alice lnd_btc
-$ docker exec -i -t alice bash
+$ sudo -E docker-compose run -d --name alice lnd_btc
```

Comment:
`sudo` isn't required although it may be required depending on your OS and security configuration.

Reply:
same with the `-E` flag.  This is OS environment dependent.

Reply:
I just noticed there is also a `docker` command in that README not just `docker-compose`. The only time you dont need root when calling docker is if your user is in the `docker` group, which is a catastrophe from a security standpoint. It effectively gives your user passwordless sudo

Reply:
@tuxx42 OSX does not require `sudo` and therefore doesn't require `-E`.  Granted this is most likely because the OSX implementation uses a guest kernel.  I think we could just make a note that `sudo -E` may be required without explicitly demanding the user elevate privilage.

Reply:
agreed.